### PR TITLE
Create 2023-01-09-do-you-speak-prestashop-december-2022.md

### PR DESCRIPTION
**Pull request by l-delin**

### CODE REVIEW

**Comment by kpodemski** [comment]
Note, please wait with the publishing for the @coffrep review and schedule for this article

**Comment by l-delin** [comment]
Ping @Julievrz

File: content/news/translation/2022/do-you-speak-prestashop-december-2022.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
+---
+layout: post
+aliases: ["/news/do-you-speak-prestashop-december-2022"]
+title: "Do you speak PrestaShop? – December 2022 edition"
+subtitle: "Where you'll learn about the Project’s 2022 retrospective"
```
Comment (Julievrz) [inline content/news/translation/2022/do-you-speak-prestashop-december-2022.md]:
```suggestion
subtitle: "Where you'll learn about the project’s 2022 retrospective"
```

File: content/news/translation/2022/do-you-speak-prestashop-december-2022.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+## Project news
+
+### ➕ The PrestaShop project 2022 retrospective 🎉
+
+2023 is here, happy new year! Thank you all for your contributions. With the help of every single one of you, PrestaShop 8.0 was released on October 26, 2022.  
+
+Overall, in 2022,
+[120 contributors](https://build.prestashop-project.org/news/2022/prestashop-8-0-0-available) contributed to 1100 pull requests on the GitHub side of the Project.
+
+On the Crowdin side of the Project, 146 members translated 47,723 strings, containing 339,246 words, and approved 319,197 words. Did you know that there are more words in the PrestaShop software than George R. R. Martin’s Game of Thrones?
```
Comment (Julievrz) [inline content/news/translation/2022/do-you-speak-prestashop-december-2022.md]:
```suggestion
On the Crowdin side of the Project, 146 members translated 339,246 words and approved 319,197 words. Did you know that there are more words in the PrestaShop software than in George R. R. Martin’s Game of Thrones? 
```

(I didn't know that 😆)

File: content/news/translation/2022/do-you-speak-prestashop-december-2022.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+### ➕ The PrestaShop project 2022 retrospective 🎉
+
+2023 is here, happy new year! Thank you all for your contributions. With the help of every single one of you, PrestaShop 8.0 was released on October 26, 2022.  
+
+Overall, in 2022,
+[120 contributors](https://build.prestashop-project.org/news/2022/prestashop-8-0-0-available) contributed to 1100 pull requests on the GitHub side of the Project.
+
+On the Crowdin side of the Project, 146 members translated 47,723 strings, containing 339,246 words, and approved 319,197 words. Did you know that there are more words in the PrestaShop software than George R. R. Martin’s Game of Thrones? 
+
+April and August were the most active months, with a total of 122,405 translated words.
```
Comment (Julievrz) [inline content/news/translation/2022/do-you-speak-prestashop-december-2022.md]: 122,405 words = in April or in August? 🤔 Or both?

File: content/news/translation/2022/do-you-speak-prestashop-december-2022.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+ 
+Remember, you can see who has been contributing to our translation project thanks to the [Translators page](https://translators.prestashop.com/).
+ 
+## Complete translations
+ 
+#### Fully available languages
+ 
+At the end of December, PrestaShop 8 was fully available (= 100% translated and validated) in **11 languages**. That’s great! Thank you for your involvement. :tada:
+ 
+| Catalan | Chinese (CN) | Chinese (TW) | Czech |
```
Comment (Julievrz) [inline content/news/translation/2022/do-you-speak-prestashop-december-2022.md]: Aren't Romanian and Chinese TW missing? They're mentioned below as new languages. Also, can you put the newly translated languages in bold, please? 😇

File: content/news/translation/2022/do-you-speak-prestashop-december-2022.md (hunk 3 above)
Comment (l-delin) [reply content/news/translation/2022/do-you-speak-prestashop-december-2022.md]: both :)

**Review by Julievrz — APPROVED** [review]

File: content/news/translation/2023/do-you-speak-prestashop-december-2022.md [hunk 5]
```diff
@@ -3,7 +3,7 @@ layout: post
 aliases: ["/news/do-you-speak-prestashop-december-2022"]
 title: "Do you speak PrestaShop? – December 2022 edition"
 subtitle: "Where you'll learn about the project’s 2022 retrospective"
-date: 2022-12-03
+date: 2022-12-09
```
Comment (Julievrz) [inline content/news/translation/2023/do-you-speak-prestashop-december-2022.md]:
```suggestion
date: 2023-01-09
```

File: content/news/translation/2023/do-you-speak-prestashop-december-2022.md [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+
+## Let’s keep in touch!
+
+If you have not joined us on Crowdin yet, [it is never too late](https://crowdin.com/project/prestashop-official)! :wink:
+
+You can also join the #translation channel of the PrestaShop Open Source Slack. On this channel, I share information about the translation project. You will also be informed about the publication of the Crowdin monthly report. Of course, you will also be able to ask questions, suggest ideas, or get in touch with other translators.
+
+Thanks everyone! 🙌
+
+{{< cta "https://join.slack.com/t/prestashop/shared_invite/zt-dkmbz5qf-I~FlEWwmRUOXunc5ui0Ucg" >}}Join us on Slack{{< /cta >}}
```
Comment (kpodemski) [inline content/news/translation/2023/do-you-speak-prestashop-december-2022.md]:
```suggestion
{{< cta "https://www.prestashop-project.org/slack/" >}}Join us on Slack{{< /cta >}}
```

File: content/news/translation/2023/do-you-speak-prestashop-december-2022.md [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+
+Overall, in 2022,
+[120 contributors](https://build.prestashop-project.org/news/2022/prestashop-8-0-0-available) contributed to 1100 pull requests on the GitHub side of the Project.
+
+On the Crowdin side of the Project, 146 members translated 339,246 words and approved 319,197 words. Did you know that there are more words in the PrestaShop software than in George R. R. Martin’s Game of Thrones? 
+
+April and August were the most active months, with a combined total of 122,405 translated words. 
+Overall, 177 new contributors joined the Crowdin Project this year, adding up to a total of 3,658 members at the end of 2022.
+
+We warmly welcome each and every one of you. ☕️
```
Comment (kpodemski) [inline content/news/translation/2023/do-you-speak-prestashop-december-2022.md]:
```suggestion
A warm welcome to each and every one of you. ☕️
```

File: content/news/translation/2023/do-you-speak-prestashop-december-2022.md [hunk 8]
```diff
@@ -0,0 +1,151 @@
...
+
+### 💕 Many thanks to December’s newcomers
+
+Many people subscribe to the project(s) of their choice each month, but not so many start translating straight away. For that reason, we would like to thank [Monika](https://crowdin.com/profile/monikaraciunaite/activity)  who rushed into the efforts immediately!
```
Comment (kpodemski) [inline content/news/translation/2023/do-you-speak-prestashop-december-2022.md]:
```suggestion
Many people subscribe to the project(s) of their choice each month, but not so many start translating straight away. For that reason, I would like to thank [Monika](https://crowdin.com/profile/monikaraciunaite/activity)  who rushed into the efforts immediately!
```

File: content/news/translation/2023/do-you-speak-prestashop-december-2022.md [hunk 9]
```diff
@@ -0,0 +1,151 @@
...
+
+#### Top 3 proofreaders in December 2022:
+ 
+| | Proofreader | Language | # Words
+|-| ---------- | -------- | ----------------
+| 1. | SNor42 | Hungarian | 972
+| 2. | danielshapiro9 | Hebrew | 161
+| 3. | M. Mirena (mmirena) | Albanian | 58
+
+Congratulations, and welcome to our new top contributors! :clap:
```
Comment (kpodemski) [inline content/news/translation/2023/do-you-speak-prestashop-december-2022.md]:
```suggestion
Congratulations, and welcome to the project's new top contributors! :clap:
```

File: content/news/translation/2023/do-you-speak-prestashop-december-2022.md (hunk 6 above)
Comment (l-delin) [reply content/news/translation/2023/do-you-speak-prestashop-december-2022.md]: Hello, this is the direct link to the translation channel :) Should I update it?

Comment (kpodemski) [reply content/news/translation/2023/do-you-speak-prestashop-december-2022.md]: This invitation link is no longer valid and requires login via google/apple or @prestashop.com email

**Comment by l-delin** [comment]
Thank you for your review @kpodemski :-)

File: content/news/translation/2023/do-you-speak-prestashop-december-2022.md [hunk 10]
```diff
@@ -0,0 +1,151 @@
...
+![Crowdin Monthly banner](/assets/images/2019/01/Build-Crowdin-banner.jpg)
+
+## Project news
+
+### ➕ The PrestaShop project 2022 retrospective 🎉
+
+2023 is here, happy new year! Thank you all for your contributions. With the help of every single one of you, PrestaShop 8.0 was released on October 26, 2022.  
+
+Overall, in 2022,
+[120 contributors](https://build.prestashop-project.org/news/2022/prestashop-8-0-0-available) contributed to 1100 pull requests on the GitHub side of the Project.
```
Comment (coffrep) [inline content/news/translation/2023/do-you-speak-prestashop-december-2022.md]:
```suggestion
[120 contributors](https://build.prestashop-project.org/news/2022/prestashop-8-0-0-available) contributed to 1100 pull requests on the GitHub side of the project.
```

File: content/news/translation/2023/do-you-speak-prestashop-december-2022.md [hunk 11]
```diff
@@ -0,0 +1,151 @@
...
+
+On the Crowdin side of the Project, 146 members translated 339,246 words and approved 319,197 words. Did you know that there are more words in the PrestaShop software than in George R. R. Martin’s Game of Thrones?
```
Comment (coffrep) [inline content/news/translation/2023/do-you-speak-prestashop-december-2022.md]:
```suggestion
On the Crowdin side of the project, 146 members translated 339,246 words and approved 319,197 words. Did you know that there are more words in the PrestaShop software than in George R. R. Martin’s Game of Thrones? 
```

File: content/news/translation/2023/do-you-speak-prestashop-december-2022.md [hunk 12]
```diff
@@ -0,0 +1,151 @@
...
+
+2023 is here, happy new year! Thank you all for your contributions. With the help of every single one of you, PrestaShop 8.0 was released on October 26, 2022.  
+
+Overall, in 2022,
+[120 contributors](https://build.prestashop-project.org/news/2022/prestashop-8-0-0-available) contributed to 1100 pull requests on the GitHub side of the Project.
+
+On the Crowdin side of the Project, 146 members translated 339,246 words and approved 319,197 words. Did you know that there are more words in the PrestaShop software than in George R. R. Martin’s Game of Thrones? 
+
+April and August were the most active months, with a combined total of 122,405 translated words. 
+Overall, 177 new contributors joined the Crowdin Project this year, adding up to a total of 3,658 members at the end of 2022.
```
Comment (coffrep) [inline content/news/translation/2023/do-you-speak-prestashop-december-2022.md]:
```suggestion
Overall, 177 new contributors joined the Crowdin project this year, adding up to a total of 3,658 members at the end of 2022.
```

File: content/news/translation/2023/do-you-speak-prestashop-december-2022.md [hunk 13]
```diff
@@ -0,0 +1,151 @@
...
+
+A warm welcome to each and every one of you. ☕️
+
+### 💕 Many thanks to December’s newcomers
+
+Many people subscribe to the project(s) of their choice each month, but not so many start translating straight away. For that reason, I would like to thank [Monika](https://crowdin.com/profile/monikaraciunaite/activity)  who rushed into the efforts immediately!
```
Comment (coffrep) [inline content/news/translation/2023/do-you-speak-prestashop-december-2022.md]:
```suggestion
Many people subscribe to the project(s) of their choice each month, but not so many start translating straight away. For that reason, I would like to thank [Monika](https://crowdin.com/profile/monikaraciunaite/activity) who rushed into the efforts immediately!
```

**Review by coffrep — APPROVED** [review]
Just added a few suggestions, notably for word consistency on the post. Thanks a lot Ldelin! And i'm approving with changes.

File: content/news/translation/2023/do-you-speak-prestashop-december-2022.md [hunk 14]
```diff
@@ -0,0 +1,151 @@
+---
+layout: post
+aliases: ["/news/do-you-speak-prestashop-december-2022"]
+title: "Do you speak PrestaShop? – December 2022 edition"
+subtitle: "Where you'll learn about the project’s 2022 retrospective"
+date: 2023-01-09
```
Comment (kpodemski) [inline content/news/translation/2023/do-you-speak-prestashop-december-2022.md]:
```suggestion
date: 2023-01-05
```

We could publish it today as the next week is something else planned.

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
Thank you @l-delin :)

**Pull request merged**